src/annotator/anchoring/pdf.ts

```diff
@@ -849,18 +858,28 @@ export async function describeShape(shape: Shape): Promise<Selector[]> {
         throw new Error('Point is not in a page');
       }
 
+      const pageView = await getPageView(point.pageIndex);
+      const [viewLeft, viewBottom, viewRight, viewTop] = pageView.pdfPage.view;
+
       return [
         {
           type: 'PageSelector',
           index: point.pageIndex,
         },
         {
           type: 'ShapeSelector',
+          anchor: 'page',
           shape: {
             type: 'point',
             x: point.x,
             y: point.y,
           },
+          view: {
+            left: viewLeft,
+            bottom: viewBottom,
+            top: viewTop,
+            right: viewRight,
+          },
```

Comment:
Maybe we can extract a helper function that converts the PDFPage view tuple into the viewport object, and remove the duplication here.